packages/plugin-commands-env/src/env.ts

```diff
@@ -24,14 +25,27 @@ export const commandNames = ['env']
 
 export function help () {
   return renderHelp({
-    description: 'Install and use the specified version of Node.js. The npm CLI bundled with the given Node.js version gets installed as well.',
+    description: 'Manage Node.js versions.',
     descriptionLists: [
+      {
+        title: 'Commands',
+        list: [
+          {
+            description: 'Installs the specified version of Node.JS. The npm CLI bundled with the given Node.js version gets installed as well.',
+            name: 'use',
+          },
+          {
+            description: 'Uninstalls the specified version of Node.JS.',
+            name: 'remove\nuninstall',
+            shortAlias: 'rm,\nun',
```

Comment:
Just write remove and rm
```suggestion
            name: 'remove',
            shortAlias: 'rm',
```